lang/csharp/src/apache/main/Schema/EnumSchema.cs

```diff
@@ -92,14 +97,18 @@ internal static EnumSchema NewInstance(JToken jtok, PropertyMap props, SchemaNam
         /// <param name="props">custom properties on this schema</param>
         /// <param name="names">list of named schema already read</param>
         /// <param name="doc">documentation for this named schema</param>
+        /// <param name="defaultToken">default token</param>
         private EnumSchema(SchemaName name, IList<SchemaName> aliases, List<string> symbols,
                             IDictionary<String, int> symbolMap, PropertyMap props, SchemaNames names,
-                            string doc)
+                            string doc, string defaultToken)
                             : base(Type.Enumeration, name, aliases, props, names, doc)
         {
             if (null == name.Name) throw new SchemaParseException("name cannot be null for enum schema.");
             this.Symbols = symbols;
             this.symbolMap = symbolMap;
+
+            if (defaultToken != null && !symbolMap.ContainsKey(defaultToken)) throw new SchemaParseException($"Default symbol: {defaultToken} not found in symbols");
```

Comment:
nit: line's getting long, would put throw on new line, and consistent with rest of codebase, though not the case immediately above ...